src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,17 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        if (value == 0) {
+            return "0";
+        }
+
+        int b;
+        StringBuilder temp = new StringBuilder("");
+        while (value != 0) {
+            b = value % 2;
+            temp.insert(0,b);
```

Comment:
try to use append()

Reply:
it'll makes your code more understandable

Reply:
Done, thanks)